app/views/userAccount.scala.html

```diff
@@ -40,7 +40,7 @@ <h3 class="header orange-text">Passkeys</h3>
                             <p>Passkey: @{passkey.name}</p>
                             <p>Added: @{dateFormat(passkey.registrationTime)}</p>
                             <p>Type: @{passkey.aaguid}</p>
-                            <div><button id="delete-passkey" class="btn" csrf-token="@{CSRF.getToken.get.value}" data-passkey="@{passkey.id}">Delete</button></div>
+                            <div><button class="btn delete-passkey-btn" csrf-token="@{CSRF.getToken.get.value}" data-passkey-id="@{passkey.id}" data-passkey-name="@{passkey.name}">Delete</button></div>
```

Comment:
passkey-name looks redundant here now:
```suggestion
                            <div><button class="btn delete-passkey-btn" csrf-token="@{CSRF.getToken.get.value}" data-passkey-id="@{passkey.id}">Delete</button></div>
```

Reply:
We actually need this attribute in the delete method for the confirmation text.